k8s/cert-manager/Makefile

```diff
@@ -8,11 +8,10 @@ VERIFY_WAIT_TIMEOUT = 1200
 CHART_NAME := cert-manager
 APP_ID ?= $(CHART_NAME)
 
-SOURCE_REGISTRY ?= marketplace.gcr.io/google
```

Comment:
Revert this file if it is not related to the kalm deprecation

Reply:
Done